### ISSUE TITLE
fail to startTest Error appear when launching new difldo report 

### ISSUE THREAD

**Issue by itay85** [Top-Q/jsystem#270]
when running the new difldo report an exception appear 

 [jsystem] יונ 07, 2016 6:16:14 PM jsystem.framework.report.RunnerListenersManager fireTestStart
  [jsystem] SEVERE: Fail to startTest
  [jsystem] java.lang.ArrayIndexOutOfBoundsException: 1
  [jsystem] 	at jsystem.extensions.report.difido.AbstractHtmlReporter.startTest(AbstractHtmlReporter.java:345)
  [jsystem] 	at jsystem.framework.report.RunnerListenersManager.fireTestStart(RunnerListenersManager.java:793)
  [jsystem] 	at jsystem.framework.report.RunnerListenersManager.startTest(RunnerListenersManager.java:772)
  [jsystem] 	at junit.framework.JSystemJUnit4ClassRunner$TestListenerAdapter.testStarted(JSystemJUnit4ClassRunner.java:174)
  [jsystem] 	at org.junit.runner.notification.RunNotifier$3.notifyListener(RunNotifier.java:87)
  [jsystem] 	at org.junit.runner.notification.RunNotifier$SafeNotifier.run(RunNotifier.java:41)
  [jsystem] 	at org.junit.runner.notification.RunNotifier.fireTestStarted(RunNotifier.java:84)
  [jsystem] 	at org.junit.internal.runners.MethodRoadie.run(MethodRoadie.java:43)
  [jsystem] 	at org.junit.internal.runners.JUnit4ClassRunner.invokeTestMethod(JUnit4ClassRunner.java:98)
  [jsystem] 	at org.junit.internal.runners.JUnit4ClassRunner.runMethods(JUnit4ClassRunner.java:61)
  [jsystem] 	at org.junit.internal.runners.JUnit4ClassRunner$1.run(JUnit4ClassRunner.java:54)
  [jsystem] 	at org.junit.internal.runners.ClassRoadie.runUnprotected(ClassRoadie.java:34)
  [jsystem] 	at org.junit.internal.runners.ClassRoadie.runProtected(ClassRoadie.java:44)
  [jsystem] 	at org.junit.internal.runners.JUnit4ClassRunner.run(JUnit4ClassRunner.java:52)
  [jsystem] 	at junit.framework.JSystemJUnit4ClassRunner.run(JSystemJUnit4ClassRunner.java:250)
  [jsystem] 	at junit.framework.JUnit4TestAdapterForJSystem.run(JUnit4TestAdapterForJSystem.java:86)
  [jsystem] 	at com.aqua.anttask.jsystem.JUnitTestRunner.run(JUnitTestRunner.java:461)
  [jsystem] 	at com.aqua.anttask.jsystem.JSystemTask.executeInVM(JSystemTask.java:1388)
  [jsystem] 	at com.aqua.anttask.jsystem.JSystemTask.execute(JSystemTask.java:957)
  [jsystem] 	at com.aqua.anttask.jsystem.JSystemTask.executeOrQueue(JSystemTask.java:1806)
  [jsystem] 	at com.aqua.anttask.jsystem.JSystemTask.execute(JSystemTask.java:897)
  [jsystem] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:288)
  [jsystem] 	at sun.reflect.GeneratedMethodAccessor141.invoke(Unknown Source)
  [jsystem] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  [jsystem] 	at java.lang.reflect.Method.invoke(Method.java:497)
  [jsystem] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:105)
  [jsystem] 	at org.apache.tools.ant.Task.perform(Task.java:348)
  [jsystem] 	at org.apache.tools.ant.Target.execute(Target.java:357)
  [jsystem] 	at org.apache.tools.ant.Target.performTasks(Target.java:385)
  [jsystem] 	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1329)
  [jsystem] 	at org.apache.tools.ant.helper.SingleCheckExecutor.executeTargets(SingleCheckExecutor.java:38)
  [jsystem] 	at org.apache.tools.ant.Project.executeTargets(Project.java:1181)
  [jsystem] 	at org.apache.tools.ant.taskdefs.Ant.execute(Ant.java:416)
  [jsystem] 	at net.sf.antcontrib.logic.AntCallBack.execute(AntCallBack.java:54)
  [jsystem] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:288)
  [jsystem] 	at sun.reflect.GeneratedMethodAccessor141.invoke(Unknown Source)
  [jsystem] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  [jsystem] 	at java.lang.reflect.Method.invoke(Method.java:497)
  [jsystem] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:105)
  [jsystem] 	at org.apache.tools.ant.Task.perform(Task.java:348)
  [jsystem] 	at org.apache.tools.ant.Target.execute(Target.java:357)
  [jsystem] 	at org.apache.tools.ant.Target.performTasks(Target.java:385)
  [jsystem] 	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1329)
  [jsystem] 	at org.apache.tools.ant.Project.executeTarget(Project.java:1298)
  [jsystem] 	at org.apache.tools.ant.helper.DefaultExecutor.executeTargets(DefaultExecutor.java:41)
  [jsystem] 	at org.apache.tools.ant.Project.executeTargets(Project.java:1181)
  [jsystem] 	at org.apache.tools.ant.Main.runBuild(Main.java:698)
  [jsystem] 	at org.apache.tools.ant.Main.startAnt(Main.java:199)
  [jsystem] 	at org.apache.tools.ant.launch.Launcher.run(Launcher.java:257)
  [jsystem] 	at org.apache.tools.ant.launch.Launcher.main(Launcher.java:104)
  [jsystem]

**Comment by itaiag** [comment]
It's hard to follow the exception since I don't know which version of JSystem is used. 
Please provide the version and, if possible, steps to reproduce the problem.

**Comment by itay85** [comment]
6.1.0.3, also tried 6.1.0.4-SNAPSHOT 
as for steps i don't really know what is the cause , I did saw that this occur only in specific code which I ran and happened when i change jsystem.properties a bit. 

On other hand yesterday I took a clean 6.1.0.4-SNAPSHOT , didn't changed the configuration at all loaded my project and it appear to be working fine now, just the report colors are gone in the 6.1.04 version. 
Thanks.
